### PR TITLE
Added async

### PR DESCRIPTION
**Pull request by VeerMuchandi**
Added async for cluster and database creation so that the wait time can be used to spin up other services.

### CODE REVIEW

File: labs/spring-boot/setup.sh [hunk 1]
```diff
@@ -56,7 +56,17 @@ gcloud beta sql instances create $DB_INSTANCE_NAME \
     --cpu=2 \
     --memory=4GB \
     --region=$REGION \
-    --root-password=${DB_INSTANCE_PASSWORD}
+    --root-password=${DB_INSTANCE_PASSWORD}\
```
Comment (gitrey) [inline labs/spring-boot/setup.sh]: might need a space before '\\'

File: labs/spring-boot/setup.sh [hunk 2]
```diff
@@ -56,7 +56,17 @@ gcloud beta sql instances create $DB_INSTANCE_NAME \
...
+    --async
+
+echo "Checking database readiness"
+while [ $(gcloud sql instances list --filter="name=quote-db-instance" --format="value(STATUS)") != "RUNNABLE" ]
+do
+  echo "Waiting for database to be ready"
+  sleep 1
```
Comment (gitrey) [inline labs/spring-boot/setup.sh]: can prob bump this to 60 seconds?

File: labs/spring-boot/setup.sh [hunk 3]
```diff
@@ -80,6 +90,14 @@ gcloud projects add-iam-policy-binding $PROJECT_ID \
   --member="serviceAccount:gke-quotedb-service-account@$PROJECT_ID.iam.gserviceaccount.com" \
   --role="roles/cloudsql.client"
 
+echo "Checking GKE clustering readiness"
+while [ $(gcloud container clusters list --filter="name=quote-cluster" --format="value(status)") == "PROVISIONING" ]
+do
+  echo "Waiting for GKE cluster to be ready"
+  sleep 1
```
Comment (gitrey) [inline labs/spring-boot/setup.sh]: can prob bump this to 60 seconds?

File: labs/spring-boot/setup.sh (hunk 1 above)
Comment (VeerMuchandi) [reply labs/spring-boot/setup.sh]: check it now

File: labs/spring-boot/setup.sh (hunk 2 above)
Comment (VeerMuchandi) [reply labs/spring-boot/setup.sh]: check it now

File: labs/spring-boot/setup.sh (hunk 3 above)
Comment (VeerMuchandi) [reply labs/spring-boot/setup.sh]: check it now

**Review by gitrey — APPROVED** [review]
Executed setup.sh in qwiklabs env and LGTM

**Pull request merged**